scripts/src/main/resources/scripts/command/kotlin_native

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup version help"
+  fi
+  exit
+fi
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+TOOL_VERSION_COMMAND="kotlinc-native -version"
```

Comment:
if the executable command is `kotlinc-native` we should rename this commandlet from `kotlin_native` to `kotlinc-native`.